### PR TITLE
Freedom

### LINKED ISSUE #383: Add MVP of culture section to project time
This will likely include examples of scenarios we have run into previously. This should include right and wrong ways to approach different situations.

One example of this could be holding up energizers on a single hacker who is not ready to go to energizers.

"I'm confused about what I'm supposed to do , I don't know where the line is, what I'm supposed to say. @gemmabusoni." Getting people to do stuff, where is the line.

hackEDU is going to do it's own thing, feel free to do the thing yourself.

### PR DESCRIPTION
**Pull request by jonleung**
Closes #383 in describing how to get someone to do something you think is good for them. This addresses the problem of what kind of freedom should exist at Hack Camp and how we might communicate this to the Hackers.

This also specifically suggest some examples of things you could say in certain scenarios, and things you probably don't want to say.

### CODE REVIEW

**Comment by maxwofford** [comment]
If this is ready to be reviewed, feel free to assign me.

**Comment by jonleung** [comment]
Hey @gemmabusoni @ssun098 @saescapa, on Friday we discussed how to communicate / "do" freedom at Hack Camp, here is an MVP of the guide for how to do that. Please let me know if you have any questions!

**Comment by maxwofford** [comment]
@jonleung Did you mean to include the messenger workshop in these commits?

**Comment by maxwofford** [comment]
@jonleung Would you please check out my latest commit to make sure it is in the spirit of the PR?

**Comment by zachlatta** [comment]
Any updates @jonleung?

On Mon, Aug 10, 2015 at 10:21 AM, Max Wofford notifications@github.com
wrote:

> @jonleung https://github.com/jonleung Would you please check out my
> latest commit to make sure it is in the spirit of the PR?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/pull/419#issuecomment-129534359.

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by jonleung** [comment]
Re: @MaxWofford 

> @jonleung Did you mean to include the messenger workshop in these commits?
> I did not intend to include the messenger workshop in my commits. What should I do now?

I also left some comments in the commit.

**Comment by maxwofford** [comment]
@jonleung Will go back to this PR on Monday

**Comment by maxwofford** [comment]
@jonleung Would you please take care of the merge conflicts. I can't tell what changes are the most recent.

**Comment by maxwofford** [comment]
@jonleung Gentle nudge on this.

File: cohort_4/playbook/workshops/messenger/README.md [hunk 1]
```diff
@@ -0,0 +1,1170 @@
...
+    var usernameInputValue = usernameInput.value;
+
+    var newLi = document.createElement("li");
+    newLi.innerHTML = usernameInputValue + " : " + messageInputValue;
+
+    var messageList = document.getElementById("messageList");
+    messageList.appendChild(messageElement);
+  }
+}
+```
```
Comment (zachlatta) [inline cohort_4/playbook/workshops/messenger/README.md:1170]: Does it make sense to include this file in this pull request?

**Comment by zachlatta** [comment]
I see two workshop commits in this pull request. Was that intentional?

File: cohort_4/playbook/workshops/messenger/README.md (hunk 1 above)
Comment (jonleung) [inline cohort_4/playbook/workshops/messenger/README.md:1170]: It doesn't... @zachlatta, how do I remove this? I'm not sure what to google, things I've thought of are:
- `removing a file from history`
- `removing accidental commit of file`

I'm not quite sure I understand the problem... I'm not sure how the messenger workshop got here...

**Comment by maxwofford** [comment]
> how do I remove this? I'm not sure what to google

@jonleung Here's how I would do it. It's probably wrong. It's probably terrible. It's straight up rewriting history, but this is what I would do.

You can change history by rebasing 1 commit past the earliest commit you are playing with: 

`git rebase -i 009c9394b21a7553773fb11706e5a2bf4704403a^1` (I aliased this to `gr -i`)

Then change the order of the commits so the messenger workshop commits(009c939 & e77f349) are at the bottom:

```
  6 pick 8b7c27c cohort 4: [freedom] write description of how to verbally address implementation of freedom
  5 pick 605a5f3 cohort 4: [philsophy] add specific example
  4 pick af8f781 cohort 4: [philosophy] syntax and grammar changes
  3 pick 009c939 cohort 4: [messenger workshop] add initial draft and outline
  2 pick e77f349 cohort 4: [workshop 4] finish mvp of workshop
```

Now we can drop the workshop commits with `git reset HEAD~2 --mixed` (I aliased this to `undo`). Let's do a quick `git status` (I aliased this to `gs`)

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

        playbook/workshops/messenger/

nothing added to commit but untracked files present (use "git add" to track)
```

Good! Our messenger workshop changes are sitting in the directory, but not committed.

Let's do a `git log` (I aliased this to `gl`)

```
commit 85fc551ca7d1a5ec35786d98c3a4cac8b7f8f9cf
Author: Max Wofford <max@hackedu.us>
Date:   Mon Aug 10 12:13:33 2015 -0700

    cohort 4: [philosophy] syntax and grammar changes

commit adde0fdc01af54f8c217c03b03d091abc6478351
Author: Jonathan Leung <me@jonl.org>
Date:   Mon Aug 10 02:48:54 2015 -0700

    cohort 4: [philsophy] add specific example

commit 116ce000e808f95345ec3d9b0cdf49c5ea8cb0c1
Author: Jonathan Leung <me@jonl.org>
Date:   Mon Aug 10 02:20:58 2015 -0700

    cohort 4: [freedom] write description of how to verbally address implementation of freedom
```

AWESOME! Force push!



---

@zachlatta How should I actually be doing this?

**Comment by zachlatta** [comment]
I'd say trying that is worth a shot. I'm honestly not good enough at git to say what should be done definitively.

If that doesn't work, perhaps do that and then rebase master into the PR's branch?

**Comment by jonleung** [comment]
I've decided to make another pull request instead of trying re-write history:

https://github.com/hackedu/hack-camp/compare/freedom_2

**Pull request closed without merging**